tests/tests_args_parser.c

```diff
@@ -0,0 +1,93 @@
+#define MUNIT_ENABLE_ASSERT_ALIASES
+#include "munit/munit.h"
+#include "args_parser.h"
+#include "vm/lib/tests_helper.h"
+#include "vm/lib/vm_string.h"
+#include "stdio.h"
```

Comment:
Remember what we said about the order of include? Also, there has to be a newline separating C's runtime include files from ours.

Reply:
Well reminded!